README.md

```diff
@@ -502,6 +502,69 @@ Enable or disable the letsencrypt plugin for a dokku app
     state: absent
 ```
 
+### dokku_network
+
+Create or destroy container networks for dokku apps
+
+#### Parameters
+
+|Parameter|Choices/Defaults|Comments|
+|---------|----------------|--------|
+|name<br /><sup>*required*</sup>||The name of the network|
+|state|*Choices:* <ul><li>**present** (default)</li><li>absent</li></ul>|The state of the network|
+
+#### Example
+
+```yaml
+- name: Create a network
+  dokku_network:
+    name: example-network
+
+- name: Delete that network
+  dokku_network:
+    app: example-network
+    state: absent
+```
+
+### dokku_network_property
+
+Set or clear a network property for a given dokku application
+
+#### Parameters
+
+|Parameter|Choices/Defaults|Comments|
+|---------|----------------|--------|
+|app<br /><sup>*required*</sup>||The name of the app|
+|network<br /><sup>*required*</sup>||The name of the network to attach to|
+|property<br /><sup>*required*</sup>|*Choices:* <ul><li>initial-network</li><li>attach-post-create</li><li>attach-post-deploy</li></ul>|The attach network property|
+
+#### Example
+
+```yaml
+- name: Associates a network after a container is created but before it is started
+  dokku_network_property:
+    app: hello-world
+    network: example-network
+    property: attach-post-create
+
+- name: Associates the network at container creation
+  dokku_network_property:
+    app: hello-world
+    network: example-network
+    property: initial-network
+
+- name: Setting a global network property
+  dokku_network_property:
+    app: --global
```

Comment:
This seems a bit hackish, maybe have a look at `dokku_domains` and how this switch is handled there?